blog/2020-05-16-web-support.md

```diff
@@ -0,0 +1,107 @@
+---
+title: React Navigation on the Web
+author: Satyajit Sahoo
+author_url: https://twitter.com/satya164
+author_title: Core Team
+author_image_url: https://avatars2.githubusercontent.com/u/1174278?s=200&v=4
+tags: [announcement]
```

Comment:
that's great! 

since this is the first blog post about `web`, what do you think about the include the tag `web` too?